client/src/components/LoginSession/LoginSession.js

```diff
@@ -1,87 +1,42 @@
-/* global window */
-import React, { useState } from 'react';
-import confirm from '@silverstripe/reactstrap-confirm';
-import Config from 'lib/Config'; // eslint-disable-line
-import backend from 'lib/Backend';
-import moment from 'moment';
 import i18n from 'i18n';
 import PropTypes from 'prop-types';
-import jQuery from 'jquery';
-import 'regenerator-runtime/runtime';
+import React, { useState } from 'react';
+import moment from 'moment';
+import confirm from '@silverstripe/reactstrap-confirm';
+import Button from 'components/Button/Button';
 
 function LoginSession(props) {
     const [loading, setLoading] = useState({ complete: false, failed: false, submitting: false });
 
-    async function logOut() {
+    function beforeRequest() {
         setLoading({ ...loading, submitting: true });
+    }
 
-        const endpoint = backend.createEndpointFetcher({
-            url: `${props.LogOutEndpoint}/:id`,
-            method: 'delete',
-            payloadSchema: {
-                id: { urlReplacement: ':id', remove: true },
-                SecurityID: { querystring: true }
-            }
-        });
-
-        endpoint({
-            id: props.ID,
-            SecurityID: Config.get('SecurityID')
-        })
-            .then(response => {
-                setLoading({
-                    complete: true,
-                    failed: !!response.error && !response.success,
-                    submitting: false
-                });
-
-                if (response.success) {
-                    setTimeout(() => {
-                        setLoading({
-                            complete: true,
-                            failed: !!response.error && !response.success,
-                            fadeOutComplete: true,
-                            submitting: false
-                        });
-
-                        jQuery.noticeAdd({
-                            text: i18n._t(
-                                'SessionManager.LOG_OUT_CONFIRMED',
-                                'Successfully logged out of device.'
-                            ),
-                            stay: false,
-                            type: 'success'
-                        });
-                    }, 2000);
-                }
-            })
-            .catch((error) => {
-                setLoading({ complete: true, failed: true, submitting: false });
+    function receivedResponse(failed) {
+        setLoading({ complete: true, failed, submitting: false });
+    }
 
-                error.response.json().then(response => {
-                    jQuery.noticeAdd({ text: response.errors, stay: false, type: 'error' });
-                });
-            });
+    function caughtException() {
+        setLoading({ complete: true, failed: true, submitting: false });
     }
 
+    // This is an async function because 'confirm' requires it
+    // https://www.npmjs.com/package/@silverstripe/reactstrap-confirm
     async function attemptLogOut() {
```

Comment:
I needs to be an function async.

See the comment above

```
    // This is an async function because 'confirm' requires it
    // https://www.npmjs.com/package/@silverstripe/reactstrap-confirm
```